charts/dbp-moodle/scripts/hook/moodle-update-preparation.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+get_current_deployment_image() {
+    kubectl get "deploy/{{ .Release.Name }}" -n "{{ .Release.Namespace }}" -o jsonpath='{..image}' |\
+        tr -s '[:space:]' '\n' |\
+        grep '{{- .Values.moodle.image.repository -}}'
+}
+
+printf "Checking if update preparations are needed\n"
+
+new_image="{{- .Values.moodle.image.registry -}}/{{- .Values.moodle.image.repository -}}:{{- .Values.moodle.image.tag -}}"
+cur_image="$(get_current_deployment_image)"
+
+printf 'Comparing old image "%s" against new image "%s"\n' "$cur_image" "$new_image"
+
+if [ "$new_image" = "$cur_image" ]; then
+    printf 'No update taking place, no preparations needed\n'
+    exit
+fi
+printf 'Image change detected\n'
+
+printf 'Disabling regular cronjob to prevent failing runs\n'
+kubectl patch cronjobs "{{ .Release.Name }}"-moodlecronjob-php-script -n "{{ .Release.Namespace }}" -p '{"spec" : {"suspend" : true }}'
```

Comment:
use helm for job name

```suggestion
kubectl patch cronjobs "{{ .Release.Name }}"-moodlecronjob-php-script -n "{{ .Release.Namespace }}" -p '{"spec" : {"suspend" : true }}'
```